eng/Signing.props

```diff
@@ -8,23 +8,28 @@
     <AllowEmptySignList Condition="'$(SignFinalPackages)' != 'true'">true</AllowEmptySignList>
   </PropertyGroup>
 
-  <Target Name="PrepareItemsToSign" BeforeTargets="Sign">
+  <ItemGroup>
+    <!--
+      Replace the default items to sign with the specific set we want. This allows the build to call
```

Comment:
This comment probably isn't relevant anymore since this isn't being called in a target.